packages/api/cms-api/package.json

```diff
@@ -132,10 +132,10 @@
         "@mikro-orm/migrations": "^5.8.4",
         "@mikro-orm/nestjs": "^5.1.0",
         "@mikro-orm/postgresql": "^5.8.4",
-        "@nestjs/common": "^9.0.0",
-        "@nestjs/core": "^9.0.0",
+        "@nestjs/common": "^9.4.3",
+        "@nestjs/core": "^9.4.3",
         "@nestjs/graphql": "^10.0.0",
-        "@nestjs/platform-express": "^9.0.0",
+        "@nestjs/platform-express": "^9.4.3",
```

Comment:
I don't think it's necessary to bump the peer dependency (which is technically) a breaking change. Could you please revert this? Updating Demo alone should be enough

Reply:
It was necessary, otherwise I became an error from nestjs that different versions were installed.
<img width="786" alt="Screenshot 2024-07-30 at 08 25 48" src="https://github.com/user-attachments/assets/996a1694-8199-4a37-ab5c-bde43eea62fa">

But with updating and afterwards reverting, the pnpm-lock.yaml updated the nestjs version and now it is no problem anymore. And as far as I understand that, it shouldn't be a problem for projects.

Reply:
This was probably due to different dev dependencies in cms-api. But as it's fixed now, nevermind.